reviews/EJOR/02/ResponsetoReviewers.tex

```diff
@@ -0,0 +1,102 @@
+    We would like to point out that this paper does provide theoretical results
+    and insights.
+    Section 3 provides detailed theoretical expressions of the queueing model's
+    performance measures.
+    Expressions of the waiting time, blocking time and proportion of
+    individuals within target have been derived and are presented in the paper.
+    Also, section 4.2 contains a list of applied scenarios where insights 
+    are presented.
+    A busy model is examined and insights on ways to increase the relative
+    efficiency of the model are presented.
+
+    Secondly here is a list of papers already published in EJOR that introduce
+    a new model and give details on numerical procedures for solving it but
+    without any new theorems:
+    \begin{itemize}
+        \item A queueing model for general group screening policies and dynamic
+        item arrivals \\
+        https://doi.org/10.1016/j.ejor.2010.05.042
+        \item Selfish routing in public services: \\
+        https://doi.org/10.1016/j.ejor.2013.04.003
+        \item The effect of ambulance relocations on the performance of
+        ambulance service providers \\
+        https://doi.org/10.1016/j.ejor.2015.12.022
+        \item Robust and stochastic formulations for ambulance deployment and
+        dispatch \\
+        https://doi.org/10.1016/j.ejor.2019.05.011
+        \item A Markovian queueing model for ambulance offload delays \\
+        https://doi.org/10.1016/j.ejor.2012.11.030
+        \item Models and algorithms for an integrated vessel scheduling and tug
+        assignment problem within a canal harbor \\
+        https://doi.org/10.1016/j.ejor.2021.10.037
+        \item The application of operational research to European air traffic
+        flow management - understanding the context \\
+        https://doi.org/10.1016/S0377-2217(99)00084-3
+    \end{itemize}
+    
+    Finally, note that the aims and scope of EJOR do not emphasise that a paper
+    should present a new theorem in order to be published.
+    Here's the aim and scope of EJOR as listed in their website: \\[0.5cm]
+    \textit{
+        The European Journal of Operational Research (EJOR) publishes high
+        quality, original papers that contribute to the methodology of operational
+        research (OR) and to the practice of decision making. EJOR contains the
+        following types of papers:
+        \begin{itemize}
+            \item Invited Reviews, explaining to the general OR audience the developments in an OR topic over the recent years
+            \item Innovative Applications of OR, describing novel ways to solve real problems
+            \item Theory and Methodology Papers, presenting original research results contributing to the methodology of OR and to its theoretical foundations,
+            \item Short Communications, if they correct important errors found in papers previously published in EJOR
+        \end{itemize}
+    }
+
```

Comment:
I would add reviewer 3's one line comment and just say "Thank you" as a response.